packages/eslint-plugin/src/tests/rule.test.ts

```diff
@@ -0,0 +1,241 @@
+import { RuleTester } from "eslint";
+import { actionRule } from "../rules/action-rule";
+import { atomRule } from "../rules/atom-rule";
+import { reatomPrefixRule } from "../rules/reatom-prefix-rule";
+
+const tester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module'
+    }
+});
+
+tester.run('reatom/atom-rule', atomRule, {
+    valid: [
+        {
+            code: `
+            import { atom } from '@reatom/framework'
+            const countAtom = atom(0, "countAtom");
+            `
+        },
+        {
+            code: `const countAtom = atom(0);`,
```

Comment:
What this test mean? We trying to not touch `atoms`'s from different imports?

Reply:
Exactly